components/script/dom/html/htmlinputelement.rs

```diff
@@ -2131,7 +2131,7 @@ fn broadcast_radio_checked(broadcaster: &HTMLInputElement, group: Option<&Atom>)
     let form = broadcaster.form_owner();
     for r in radio_group_iter(broadcaster, group, form.as_deref(), &root) {
         if broadcaster != &*r && r.Checked() {
-            r.SetChecked(false);
+            r.SetChecked(false, CanGc::note());
```

Comment:
can you pass can_gc radio_group_updated -> broadcast_radio_checked ->SetChecked ?

Reply:
Updated.